soccer/src/soccer/strategy/agent/position/blocker.cpp

```diff
@@ -0,0 +1,19 @@
+#include "blocker.hpp"
+
+namespace strategy {
+Blocker::Blocker(double factor) : factor{factor} {}
+std::optional<RobotIntent> Blocker::get_task(RobotIntent intent,
+                                             const WorldState* const world_state) {
+    // face ball
+    planning::PathTargetFaceOption face_option = planning::FaceBall{};
+    // avoid ball
+    constexpr bool ignore_ball = false;
+    auto goal_center = rj_geometry::Point{0.0, 0.0};  // TODO: replace with field coordinates
+    planning::LinearMotionInstant goal{
+        (world_state->ball.position - goal_center) * factor + goal_center,
```

Comment:
so right now it looks like the blocker has to block between the goal and the ball, which is the same behavior as the waller. can you change this to block between any two arbitrary points (e.g. two robots' positions)?